tools/cli/internal/openapi/openapi.go

```diff
@@ -97,11 +97,19 @@ func NewChangelog(base, revision, exceptionFilePath string) (*Changelog, error)
 
 	changelogConfig := checker.NewConfig(
 		checker.GetAllChecks()).WithOptionalChecks(breakingChangesAdditionalCheckers).WithDeprecation(deprecationDaysBeta, deprecationDaysStable)
+
 	return &Changelog{
 		Base:              baseSpec,
 		Revision:          revisionSpec,
 		ExceptionFilePath: exceptionFilePath,
 		Config:            changelogConfig,
+		OasDiff: &OasDiff{
```

Comment:
we have also been using --flatten flag, can you check if there is a way to define this in oasdiff config? if not, we can create a ticket to investigate if we can use the flattening somehow, and i can check or we can try to contribute, but I don't think it's a blocker for this PR

Reply:
updated